src/Azs.ContainerRegistry.Admin/readme.md

```diff
@@ -0,0 +1,118 @@
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 2.2.3 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Azs.ContainerRegistry.Admin`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+## Generation Requirements
+Use of the beta version of `autorest.powershell` generator requires the following:
+- [NodeJS LTS](https://nodejs.org) (10.15.x LTS preferred)
+  - **Note**: It *will not work* with Node < 10.x. Using 11.x builds may cause issues as they may introduce instability or breaking changes.
+> If you want an easy way to install and update Node, [NVS - Node Version Switcher](../nodejs/installing-via-nvs.md) or [NVM - Node Version Manager](../nodejs/installing-via-nvm.md) is recommended.
+- [AutoRest](https://aka.ms/autorest) v3 beta <br>`npm install -g autorest@beta`<br>&nbsp;
+- PowerShell 6.0 or greater
+  - If you don't have it installed, you can use the cross-platform npm package <br>`npm install -g pwsh`<br>&nbsp;
+- .NET Core SDK 2.0 or greater
+  - If you don't have it installed, you can use the cross-platform npm package <br>`npm install -g dotnet-sdk-2.2`<br>&nbsp;
+
+## Run Generation
+In this directory, run AutoRest:
+> `autorest`
+
+---
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+  - $(this-folder)/../readme.azurestack.md
+  - $(repo)/specification/azsadmin/resource-manager/containerregistry/readme.azsautogen.md
+
+metadata:
+  description: 'Microsoft AzureStack PowerShell: ContainerRegistry Admin cmdlets'
+
+### PSD1 metadata changes
+subject-prefix: ''
+module-version: 1.0.0
+service-name: ContainerRegistryAdmin
+
+### File Renames
+### IMPORTANT - Note that the following settings are case sensitive ###
+module-name: Azs.ContainerRegistry.Admin
+csproj: Azs.ContainerRegistry.Admin.csproj
+psd1: Azs.ContainerRegistry.Admin.psd1
+psm1: Azs.ContainerRegistry.Admin.psm1
+```
+
+### Parameter default values
+``` yaml
+directive:
+    # Prepend ContainerRegistry for the Quota cmdlets
+  - where:
+      subject: Quota
+    set:
+      subject-prefix: ContainerRegistry
+
+    # Rename Set-AzsContainerRegistryQuota to Update-AzsContainerRegistryQuota
+  - where:
+      verb: Set
+      subject: ContainerRegistryQuota
+    set:
+      verb: Update
```

Comment:
verb Update typically implies Patch call, Set- is preferred

Reply:
We discussed with @rjaini about this renaming and agreed with this change.
@rjaini are we still ok with Update or follow @bganapa recommendation?

Reply:
@larisaborodina: Initially we thought to keep like this to be in consistent with update commands of container registry. But I remember we discussed in standup to follow Azure Stack hub guidelines as that is what customer is more familiar. So based on that we should move to Set rather update as all AZS command follow the same convention.

Reply:
Done